index.html

```diff
@@ -1,19 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
-<body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<body>
+    <header>
+        <img src="./img/karma-logo.svg" alt="Karma Logo">
+        <nav>
+            <ul>
+                <li>
+                    <a href="#"><strong>Meet Karma</strong></a>
+                </li>
+                <li>
+                    <a href="#">How it Works</a>
+                </li>
+                <li>
+                    <a href="#">Store</a>
+                </li>
+                <li>
+                    <a href="#">Blog</a>
+                </li>
+                <li>
+                    <a href="#">Help</a>
+                </li>
+                <li>
+                    <a href="#">Login</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="hamburger-menu-wrapper">
+            <div class="hamburger-menu"></div>
+            <div class="hamburger-menu"></div>
+            <div class="hamburger-menu"></div>
+        </div>
+    </header>
+    <main>
+        <section class="hero">
+            <h1>Introducing Karma</h1>
+            <p>Bring WiFi with you, everywhere you go.</p>
+            <button>Learn More</button>
+        </section>
+        <section class="karma-details">
+            <h2>Everyone needs a little Karma</h2>
+            <div class="row-container">
+                <div class="column-container">
+                    <img src="./img/icon-devices.svg" alt="Image outline of devices that can connect to the internet">
+                    <p>Internet for all devices</p>
+                </div>
+                <div class="column-container">
+                    <img src="./img/icon-coffee.svg" alt="Image outline of a steaming cup of coffee">
+                    <p>Boost your productivity</p>
+                </div>
+                <div class="column-container">
+                    <img src="./img/icon-refill.svg" alt="Image outline of a gas pump">
+                    <p>Pay as you go</p>
+                </div>
+            </div>
+        </section>
+        <section class="feature">
+            <img src="./additional/feature-img.png">
+            <div class="feature-wrapper">
```

Comment:
The class name here is confusing because usually wrapper indicates it's on the outside (like you've done above with `hamburger-wrapper` above).

Reply:
Oh yes that's true, I will rename it